graph-builder/src/status.rs

```diff
@@ -0,0 +1,62 @@
+//! Status service.
+
+use actix_web::{HttpRequest, HttpResponse};
+use futures::future;
+use futures::prelude::*;
+use prometheus;
+use std::sync::{Arc, RwLock};
+
+/// State for the status service.
+#[derive(Clone)]
+pub struct StatusState {
+    // Cached graph.
```

Comment:
Did you forget one `/` here? If not please dismiss this comment

Reply:
Indeed. Added now.